ga4gh/backend.py

```diff
@@ -404,6 +405,89 @@ def _topLevelObjectGenerator(self, request, numObjects, getByIndexMethod):
                 nextPageToken = str(currentIndex)
             yield object_.toProtocolElement(), nextPageToken
 
+    def _filterReadGroupsByBioSampleId(self, request, readGroupSet):
+        newRg = []
+        for rg in readGroupSet.readGroups:
+            print(rg.bioSampleId, request.bioSampleId)
+            if request.bioSampleId == rg.bioSampleId:
+                newRg.append(rg)
+        readGroupSet.readGroups = newRg
+        return readGroupSet
+
+    def _readGroupSetObjectGenerator(
```

Comment:
So the `readGroupSet`s returned by this generator have their `readGroup`s filtered out by `bioSampleId`?
Is this sufficiently clearly spelled out in the schemas?